cpp/src/CMakeLists.txt

```diff
@@ -83,7 +84,9 @@ install(FILES
 install(FILES
   toppra/geometric_path/piecewise_poly_path.hpp
   DESTINATION include/toppra/geometric_path)
-
+install(FILES
+  toppra/parametrizer/const_accel.hpp
```

Comment:
Didn't you forget to commit this file ?

Reply:
Please remember that this PR is related to the branch `feat-parametrizer`.
The `const_accel` files were already [there.](https://github.com/hungpham2511/toppra/tree/feat-parametrizer/cpp/src/toppra/parametrizer)

Reply:
Ok, I missed that. Thanks.

Reply:
Looking forward to find this `parametrizer` extension also in the master ;-)